source/sass/buyers-guide/views/product.scss

```diff
@@ -34,6 +34,57 @@ $pni-product-breakpoint-larger: $bp-md;
   }
 }
 
+html[lang="de"] .product-header-content.show-privacy-ding::after {
```

Comment:
Let's group the following code to within the `&:after {}` block above.

e.g., 

instead of having
```scss
html[lang="de"] .product-header-content.show-privacy-ding::after {
  background-image: url(../_images/buyers-guide/icon-privacy-ding-de.svg);
  width: 50px;
  height: 70px;

  @media (min-width: $pni-product-breakpoint-larger) {
    width: 67px;
    height: 92px;
  }
}

```

add this on line 34 instead

```scss
@at-root html[lang="de"] & {
  background-image: url(../_images/buyers-guide/icon-privacy-ding-de.svg);
  width: 50px;
  height: 70px;

  @media (min-width: $pni-product-breakpoint-larger) {
    width: 67px;
    height: 92px;
  }
}
```